### ISSUE TITLE
Status field missing from requests

### ISSUE THREAD

**Issue by Jerryescandon** [GCTC-NTGC/gc-digital-talent#4771]
## Description
The table of candidates that is the result from a request is missing the "STATUS" field as part of it.
This is an issue because only candidates with the "Qualified available" and "Placed Casual" should be returned by default but we should allow DCM to edit this if they need to.

For comparison, the candidates table in the pools does have the status field in it.

### FILTERS ON THE REQUEST PAGE

![image](https://user-images.githubusercontent.com/16977254/202218769-5b4f629c-71c7-4c4a-a3c9-ee58876a8c60.png)

### FILTERS ON VIEW CANDIDATES (FROM POOL)

![image](https://user-images.githubusercontent.com/16977254/202269237-a9ec42a5-21cc-4eea-95f7-7e769031fb5b.png)

### EDIT

🤔 There's actually more disparity in these two modals.

Missing from Requests table
- Status
- Has Diploma 
- Employment Equity
- Priority

Missing from Candidates table
- Duration preferences
- Availability
- Profile Complete (tbh we should remove this one from both)
- Skill Filter
- Government Employee

## Acceptance Criteria
- [ ] ...

**Comment by vd1992** [comment]
Requests table currently deals in Users and not Pool Candidates, it is slated to be changed in #4672

**Comment by gobyrne** [comment]
yes, let's revisit after #4672. Closing for now
